### ISSUE TITLE
Python 3 wheels for Windows x64 aren't getting built

### ISSUE THREAD

**Issue by soltanmm** [grpc/grpc#7485]
And I don't know why the Py3 builds are reported as 'succeeding' anyway on the Windows builders.

Setting for 1.1 as a long-tail target because Python 3 support is still *technically* in beta, but this issue is receiving immediate attention.

**Comment by tseaver** [comment]
The 3.4 wheels are also not installable on Python 3.5, leading to [gcloud Appveyor errors](https://ci.appveyor.com/project/dhermes/gcloud-python/build/1.0.551.master/job/h3887en1gexqdivn).

**Comment by nathanielmanistaatgoogle** [comment]
@kpayson64 spent some time on this in the last week (thanks!).

**Comment by nathanielmanistaatgoogle** [comment]
@tseaver: I'm not sure I understand [your comment](https://github.com/grpc/grpc/issues/7485#issuecomment-239233103) - are you saying that there is a circumstance in which you are able to `pip install --upgrade --pre grpcio` on Windows with Python 3.4 but not (with the same Windows and underlying system) not Python 3.5?

**Comment by tseaver** [comment]
@nathanielmanistaatgoogle The Appveyor builds succeed for Python 3.4 32-bit, but fail for Python 3.4 64-bit and for both Python 3.5 32-bit and 64-bit:  https://ci.appveyor.com/project/dhermes/gcloud-python

**Comment by nathanielmanistaatgoogle** [comment]
The 3.5 issue may be a separate problem; I've filed [issue 7725](https://github.com/grpc/grpc/issues/7725) for it.

**Comment by kpayson64** [comment]
This issue was caused by 64 bit Python3.4 not distributing the static libraries needed for mingw linking.  I've generated the needed libpython3.4.a library, and uploaded it to gcs in our grpcTesting project.  I've also updated the windows worker bootstrap script to copy this file into the Python34 installation directory as needed.

As a side note, we still need Mingw for Python3.4 because the default msvc compiler for 3.4 is still too old for our code.
